ironfish/src/wallet/wallet.test.slow.ts

```diff
@@ -1128,4 +1141,193 @@ describe('Wallet', () => {
+
+      const trustedDealerPackage: TrustedDealerKeyPackages = splitSecret(
+        coordinatorSaplingKey.spendingKey,
+        minSigners,
+        maxSigners,
+        identifiers,
+      )
+
+      const keyPackages = Object.entries(trustedDealerPackage.keyPackages)
+
+      const getMultiSigKeys = (index: number) => {
+        return {
+          identifier: keyPackages[index][0],
+          keyPackage: keyPackages[index][1],
+          proofGenerationKey: trustedDealerPackage.proofGenerationKey,
+        }
+      }
+
+      const participantA = await node.wallet.importAccount({
+        version: 2,
+        id: uuid(),
+        name: keyPackages[0][0],
+        spendingKey: null,
+        createdAt: null,
+        multiSigKeys: getMultiSigKeys(0),
+        ...trustedDealerPackage,
+      })
+      const participantB = await node.wallet.importAccount({
+        version: 2,
+        id: uuid(),
+        name: keyPackages[1][0],
+        spendingKey: null,
+        createdAt: null,
+        multiSigKeys: getMultiSigKeys(1),
+        ...trustedDealerPackage,
+      })
+      const participantC = await node.wallet.importAccount({
+        version: 2,
+        id: uuid(),
+        name: keyPackages[2][0],
+        spendingKey: null,
+        createdAt: null,
+        multiSigKeys: getMultiSigKeys(2),
+        ...trustedDealerPackage,
+      })
+
+      Assert.isNotUndefined(participantA.multiSigKeys)
+      Assert.isNotUndefined(participantB.multiSigKeys)
+      Assert.isNotUndefined(participantC.multiSigKeys)
+
+      const signingCommitments: Record<string, SigningCommitments> = {
+        [participantA.multiSigKeys.identifier]: roundOne(
+          participantA.multiSigKeys.keyPackage,
+          seed,
+        ),
+        [participantB.multiSigKeys.identifier]: roundOne(
+          participantB.multiSigKeys.keyPackage,
+          seed,
+        ),
+        [participantC.multiSigKeys.identifier]: roundOne(
+          participantC.multiSigKeys.keyPackage,
+          seed,
+        ),
+      }
+
+      // mine block to send IRON to multisig account
+      const miner = await useAccountFixture(node.wallet, 'miner')
+      const block = await useMinerBlockFixture(node.chain, undefined, miner)
+      await expect(node.chain).toAddBlock(block)
+      await node.wallet.updateHead()
+
+      const transaction = await node.wallet.send({
+        account: miner,
+        outputs: [
+          {
+            publicAddress: participantA.publicAddress,
+            amount: BigInt(2),
+            memo: '',
+            assetId: Asset.nativeId(),
+          },
+        ],
+        fee: BigInt(0),
+      })
+
+      // Create a block with a miner's fee and the transaction to send IRON to the multisig account
+      const minersfee2 = await nodeTest.strategy.createMinersFee(
+        transaction.fee(),
+        block.header.sequence + 1,
+        generateKey().spendingKey,
```

Comment:
nit - I think we could reuse the `miner` account fixture here and below instead of generating new keys

Reply:
done